Dockerfile

```diff
@@ -56,6 +56,7 @@ RUN git clone https://github.com/NVIDIA/NeMo.git && \
     git cherry-pick --no-commit -X theirs \
         fa8d416793d850f4ce56bea65e1fe28cc0d092c0 \
         a7f0bc1903493888c31436efc2452ff721fa5a67 \
+        52d50e9e09a3e636d60535fd9882f3b3f32f92ad \
```

Comment:
in addition, picking in this commit will be good https://github.com/NVIDIA/NeMo/pull/8323/files